README.md

```diff
@@ -15,16 +15,62 @@ This charm is useful when developing charms or when deploying charms in non-prod
 
 ## Usage
 
-To use the `self-signed-certificates` operator and provide certificates to your charm, your charm
-needs to support the `tls-certificates` interface.
+Bootstrap a Kubernetes (e.g. [Multipass-based MicroK8s](https://discourse.charmhub.io/t/charmed-environment-charm-dev-with-canonical-multipass/8886)) and create a new model using Juju >= 3.1:
 
-```console
-juju deploy self-signed-certificates
-juju deploy <your charm>
-juju relate self-signed-certificates <your charm>
+```shell
+juju add-model certificates
+juju deploy self-signed-certificates --channel edge
+```
+
+To confirm the deployment, you can run:
+
+```shell
+juju status --watch 1s
+```
+
+If required, you can remove the application by running:
+
+```shell
+juju remove-application self-signed-certificates
+```
+
+If required, you can remove the deployment completely by running:
+
+```shell
+juju destroy-model certificates
+```
```

Comment:
I don't think this is necessary as it is the case for all charms and not specific to this one. I'd keep the doc specific to the charm here.

Reply:
OK, I will rollback to the previous version.

Reply:
Done